src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -2931,9 +2931,10 @@ let%test_module "staged ledger tests" =
       let%bind ledger_init_state = Ledger.gen_initial_ledger_state in
       let%bind iters = Int.gen_incl 1 (max_blocks_for_coverage 0) in
       let num_cmds = transaction_capacity * iters in
+      let signature_kind = Mina_signature_kind.t_DEPRECATED in
       let%bind cmds =
-        User_command.Valid.Gen.sequence ~length:num_cmds ~sign_type:`Real
-          ledger_init_state
+        User_command.Valid.Gen.sequence ~length:num_cmds
+          ~sign_type:(`Real signature_kind) ledger_init_state
```

Comment:
```suggestion
          ~sign_type:(`Real Testnet) ledger_init_state
```

not necessary in this PR, I'm ok to leave it for later

Same for other occasions in the `src/lib/staged_ledger/staged_ledger.ml`